docs/UserGuide.md

```diff
@@ -40,147 +30,332 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
 > * Items in `SQUARE_BRACKETS` are optional.
 > * Items with `...` after them can have multiple instances.
 > * Parameters can be in any order.
+> * Options with '/' allow either word to be used.
+
+### 2.0. Viewing tasks
+Viewing all tasks in the todo list<br>
+Format: `list`
+> * A list of tasks will always be displayed.
+> * When starting the program, the list will show all tasks which are currently not completed.
+> * When performing find operations, this list will be updated to show only the results searched for.
 
 ### 2.1. Viewing help : `help`
 
 Format: `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
+> Help is also shown if you enter an incorrect command. e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task: `add`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to the todo list<br>
+Format: `add NAME [startdate/STARTDATE] [enddate/ENDDATE] [recurring/PERIOD(day/week/month)] [priority/PRIORITY] [tag/TAG]...`
 
-> Persons can have any number of tags (including 0)
+> Tasks can have any number of tags (including 0).
 
 Examples:
 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
+* `add Do Homework`
+* `add Meeting With Boss enddate/11-11-2011 19:30 startdate/11-11-2011 17:30`
+* `add Check Email startdate/today recurring/day`
 
-### 2.3. Listing all persons : `list`
+### 2.3. Updating a task : `update`
 
-Shows a list of all persons in the address book.<br>
-Format: `list`
+Updates an existing task in the todo list.<br>
+Format: `update INDEX [NAME] [startdate/STARTDATE] [enddate/ENDDATE] [recurring/PERIOD(day/week/month)] [priority/PRIORITY] [tag/TAG]...`
 
-### 2.4. Editing a person : `edit`
-
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
-
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
+> * Updates the task at the specified `INDEX`. <br>
+    The index refers to the index number shown in the last task listing.<br>
     The index **must be a positive integer** 1, 2, 3, ...
-> * At least one of the optional fields must be provided.
+> * If no optional fields are added, task will be set to completed if not completed.
+> * Otherwise, if task is completed, the task will be set to not completed.
 > * Existing values will be updated to the input values.
```

Comment:
As your parameters are optional, more thinking can be placed for this `update` function.
Maybe you can explain more, like what will happen to the existing values that are not specified in a particular `Update` command.
Eg. update 2 Assignment5
- What will happen to the dates that were earlier on but missing now?
- Or what if I intend to leave out a start time to convert it from a task that only have deadline, etc.